applications/CableNetApplication/cable_net_application_variables.cpp

```diff
@@ -0,0 +1,19 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Klaus B. Sautter
+//
+
+#include "cable_net_application_variables.h"
```

Comment:
unless you will add sth here I would remove this file
up to you

Reply:
I'll add sth. here at some point